### PR TITLE
tests:lib:at_cmd_parser: Added unit test for is_clac

### PR DESCRIPTION
**Pull request by dagbja**
NCSDK-5199: Temporary fix for failing commands (HWVERSION etc.)

Signed-off-by: Dag Bjarvin <Dag.Bjarvin@nordicsemi.no>

### CODE REVIEW

**Review by coderbyheart — APPROVED** [review]

File: tests/lib/at_cmd_parser/at_utils/src/main.c [hunk 1]
```diff
@@ -165,6 +165,19 @@ static void test_number_detection(void)
...
 }
 
+static void test_is_clac(void)
+{
+	char str[5] = "AT+ ";
+
+	zassert_true(is_clac(str), "AT+ was not detected");
+
+	str[2] = '%';
+	zassert_true (is_clac(str), "AT%  was not detected");
```
Comment (rlubos) [inline tests/lib/at_cmd_parser/at_utils/src/main.c:175]:
```suggestion
	zassert_true(is_clac(str), "AT%  was not detected");
```

File: tests/lib/at_cmd_parser/at_utils/src/main.c [hunk 2]
```diff
@@ -165,6 +165,19 @@ static void test_number_detection(void)
...
+
+	str[3] = 'X';
+	zassert_false (is_clac(str), "AT%X was detected");
```
Comment (rlubos) [inline tests/lib/at_cmd_parser/at_utils/src/main.c:178]:
```suggestion
	zassert_false(is_clac(str), "AT%X was detected");
```

**Review by rlubos — COMMENTED** [review]
Just some nits.

**Review by de-nordic — APPROVED** [review]
Looks and works OK.

Thanks!. This fixes one of the problems that have appeared with ongoing Upmerge, please merge it soon.

File: tests/lib/at_cmd_parser/at_utils/src/main.c (hunk 1 above)
Comment (dagbja) [reply tests/lib/at_cmd_parser/at_utils/src/main.c]: Fixed

File: tests/lib/at_cmd_parser/at_utils/src/main.c (hunk 2 above)
Comment (dagbja) [reply tests/lib/at_cmd_parser/at_utils/src/main.c]: Fixed

**Review by rlubos — APPROVED** [review]

**Pull request merged**